src/network/gossip.rs

```diff
@@ -273,6 +281,8 @@ impl SnapchainGossip {
         // Listen on all assigned port for this id
         swarm.listen_on(config.address.parse()?)?;
 
+        let announce_address = Self::get_announce_address(config).await;
```

Comment:
Let's log as info what we're using as the announce_address

Reply:
Addressed! https://github.com/farcasterxyz/snapchain/pull/402/commits/5641a6c019b50324da2946c46a17a03ea1b9dcba